velox/docs/configs.rst

```diff
@@ -164,4 +164,16 @@ data size exceeds this limit, otherwise it spills the partition with most data.
 If the limit is zero, then the spiller always spills a previously spilled
 partition if it has any data. This is to avoid spill from a partition with a
 small amount of data which might result in generating too many small spilled
-files.
\ No newline at end of file
+files.
+
+Spark-specific configuration
+----------------------------
+
+``legacySizeOfNull``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``bool``
+    * **Default value:** ``true``
+
+If legacySizeOfNull is set to false, the function returns null for null input.
```

Comment:
No need to repeat the property name. "If false, the functions ...". However, it is not clear which function this property applied to. Would you clarify?

Reply:
Thanks, fixed it.